src/main/java/com/google/codeu/servlets/CreateNewEventServlet.java

```diff
@@ -82,7 +82,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     setEventImageUrl(request,event);*/
     datastore.storeEvent(event);
     
-    response.sendRedirect("/user-page.html?user=" + title);
+    response.sendRedirect("/user-page.html?user=" + user);
```

Comment:
why are you sending this to user? wouldn't it make sense to take the person to the event's page after creating the event?

Reply:
Kassidy, I honestly believe you have to pay more attention what a person code. Im not the one who is in charge of the Create New Servlet, but I just wanted to fix that part of the code because what has more sense? sending to the user to user-page.html with the variable USER or with the title of the event(DOES NOT EXIST), I didnt know you want to send to the user to the event  page. So I think is better if you want change that, you have to do that. And also an advice try to test the code after doing a merge to see that kind of errors.

Reply:
Hey Kassidy, I agree that we should definitely be making multiple pull requests for the new features we add in the web app. I have been guilty of this and will try to fix this issue, as well as Eduardo. I believe you may be incorrectly interpreting this Eduardo's change to CreateNewEventServlet. Eduardo and I have been fixing a lot of Imani's errors in the servlet, and Eduardo was just fixing this redirect. Beforehand, Imani had it where it was going to an invalid redirect. It should not have been going to the title, so Eduardo fixed it. Eduardo was simply trying to fix Imani's accidental error for the time being.  here also isn't a event-page. html with all the events, unless you're referring to the home page. So, I approve this change for now as it was just fixing an error.

Reply:
I get what he's changing and why, but now I'm just suggesting change it to a better link while he's at it. It's literally just changing the link to "/event.html?event=" + event.id so I figure if he's going to change it now it would be better just to change it to that link so we don't have to change it again in the future.

Reply:
I'll just approved it for now and I'll create a new pull request if we want to change it.